projects/can_dump/scripts/dump_system.py

```diff
@@ -79,7 +78,7 @@ def parse_msg(can_id, data):
 
     msg_type_name = 'ACK' if msg_type == 1 else 'DATA'
 
-    masked = []
+    masked = [32]
```

Comment:
Magic 32?

Reply:
Removed. It's masking the battery voltage/temp message.